components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -668,6 +668,18 @@ def leave_none_unset_int(s):
          ("The error message shown to users who enter an incorrect username "
           "or password.")],
 
+    "omero.web.top_logo":
+        ["TOP_LOGO",
+         "",
+         str,
+         ("Customize the webclient top bar logo. The recommended image height "
```

Comment:
Is there a recommendation on the format of the logo URL? i.e. is it safest to use `//my.host/logo.png` to deal with both http and https deployments? Or better to suggest always using https?

Reply:
Incidentally, I was kind of surprised with my own testing that https: to a different host worked but file: URLs didn't. Maybe that was a Firefox "feature".

Reply:
I think sysadmins can figure out what's best for themselves. We have at least one other place where a url can be specified and I don't think we should add detailed basic advice to every property. The image can also be hosted on the same server in another application so the `protocol://hostname` isn't always necessary